### ISSUE TITLE
Bump Node to 16

### ISSUE THREAD

**Issue by jburgess** [AceFire6/file_changes#199]
First off, great action. Thanks so much for contributing!! Possible to bump the node version to get around the below warning?

```
Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: AceFire6/file_changes@v0.1.0
```

**Comment by AceFire6** [comment]
Hey, thanks for making the issue. I'll get this updated 👍

**Comment by AceFire6** [comment]
@jburgess this should be fixed by Release v0.2.0
